public/app/features/plugins/plugin_loader.ts

```diff
@@ -93,6 +94,11 @@ exposeToPlugin('app/features/dashboard/impression_store', {
   __esModule: true,
 });
 
+exposeToPlugin('app/core/services/backend_srv', {
```

Comment:
Just add a comment here that this is added temporarily and will likely be replaced with something officially supported in the future. But is exposed to allow plugin authors a away to make http requests without having to wait for an official (via npm package / interface) solution. Only use the get/delete/post/patch/request methods, no other methods.

Reply:
👍 sounds good